nullaway/src/main/java/com/uber/nullaway/fixserialization/FixSerializationConfig.java

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
For all copyright headers, use 2022 (won't land until then 🙂):
```suggestion
 * Copyright (c) 2022 Uber Technologies, Inc.
```

Reply:
Changed all headers. 411718dbd9893d54a75e563f7d426182d7d02334